pkg/handlers/generic/mutation/etcd/inject.go

```diff
@@ -45,6 +47,21 @@ func newEtcdPatchHandler(
 	}
 }
 
+var defaultEtcdExtraArgs = map[string]string{
+	"auto-tls":      "false",
+	"peer-auto-tls": "false",
+	"cipher-suites": strings.Join(
+		[]string{
+			tls.CipherSuiteName(tls.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256),
+			tls.CipherSuiteName(tls.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256),
+			tls.CipherSuiteName(tls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384),
+			tls.CipherSuiteName(tls.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384),
+		},
+		",",
+	),
+	"tls-min-version": "TLS1.2",
+}
+
```

Comment:
Please add a brief comment explaining the goal of this configuration (similar to my comment about the tests). :pray:

Reply:
Added.